modules/contact/src/problems/AugmentedLagrangianContactProblem.C

```diff
@@ -85,61 +84,73 @@ AugmentedLagrangianContactProblem::checkNonlinearConvergence(std::string & msg,
                                                           ref_resid,
                                                           my_div_threshold);
 
-  _console << "Augmented Lagrangian contact iteration " << _num_lagmul_iterations << std::endl;
+  _console << "Augmented Lagrangian contact iteration " << _num_lagrangian_update_iterations
+           << std::endl;
 
   bool repeat_augmented_lagrange_step = false;
 
   if (reason == MooseNonlinearConvergenceReason::CONVERGED_FNORM_ABS ||
       reason == MooseNonlinearConvergenceReason::CONVERGED_FNORM_RELATIVE ||
       reason == MooseNonlinearConvergenceReason::CONVERGED_SNORM_RELATIVE)
   {
-    if (_num_lagmul_iterations < _max_lagmul_iters)
+    if (_num_lagrangian_update_iterations < _maximum_lagrangian_update_iterations)
     {
-      NonlinearSystemBase & nonlinear_sys = getNonlinearSystemBase();
+      auto & nonlinear_sys = getNonlinearSystemBase();
```

Comment:
```suggestion
      auto & nonlinear_sys = currentNonlinearSystem();
```

Reply:
Yikes, should we remove the default argument for `sys_num`?

```
NonlinearSystemBase & getNonlinearSystemBase(unsigned int sys_num = 0);
```

Reply:
We definitely should at some point. I did the default argument so that I had some initial backwards compatibility, but yes much safer to have it removed